lib/datadog/tracing/contrib/aws/instrumentation.rb

```diff
@@ -38,7 +38,7 @@ def annotate!(span, context)
             span.set_tag(Tracing::Metadata::Ext::TAG_OPERATION, Ext::TAG_OPERATION_COMMAND)
 
             # Tag as an external peer service
-            if Contrib::SpanAttributeSchema.default_span_attribute_schema?
+            if SpanAttributeSchema.default_span_attribute_schema?
               span.set_tag(Tracing::Metadata::Ext::TAG_PEER_SERVICE, span.service)
               span.set_tag(Tracing::Metadata::Ext::TAG_PEER_HOSTNAME, context.safely(:host))
```

Comment:
Why PEER_HOSTNAME is not sent on v1?

Reply:
This is the old `peer.service` tag which is being contained solely in v0. In the future, we will set the new tag in v1